### ISSUE TITLE
Build a script to collect the github repos for CodeForPhilly

### ISSUE THREAD

**Issue by dherbst** [CodeForPhilly/cfp-data-pipeline#11]
We need the list of github repos in the following format for the tap-github:

> Update - it is space-delimited, not comma-delimited

`repository: org/repo_name user/repo_name org/repo_name`

@themightychris has pointed out we can get the list of CFP projects from https://brigade.cloud/static-api/v1/organizations/Code%20for%20Philly/projects.json

Definition of done:
- [ ] Build a script (python or whatever) to iterate through the projects and output like above: `repository: CodeForPhilly/cfp-data-pipeline`
- [ ] Have the script update the `meltano.yml` file https://github.com/CodeForPhilly/cfp-data-pipeline/blob/master/meltano/cfp-pipeline/meltano.yml#L17 to update the `repository` line.
- [ ] Bonus - build a github action to run your script

**Comment by themightychris** [comment]
The `gitsheets` npm module may make reading this data easier.

It can be invoked from CLI:

```bash
npm install -g gitsheets

git clone --depth=1 --branch=snapshot/v1 https://github.com/codeforamerica/brigade-project-index.git
cd brigade-project-index

git sheet query projects --filter.organization_name='Code for Philly' --format=json --fields=code_url
```

The node module can be used from node code to read directly too

The data also gets published to a static API you can fetch from GitHub pages:

- https://github.com/codeforamerica/brigade-project-index/tree/gh-pages/static-api/v1/organizations/Code%20for%20Philly
- https://brigade.cloud/static-api/v1/organizations/Code%20for%20Philly/projects.json

**Comment by dherbst** [comment]
Thanks @themightychris!  I updated the description so if someone wants to do this they can start with the projects.json endpoint - or any of the others you listed.

**Comment by themightychris** [comment]
Oh also, you can just hit CfP.org's laddr instance live to skip going through the global index: https://codeforphilly.org/projects?format=json (`Accept: application/json` works too instead of `?format=json` -- and this is true for nearly every page on codeforphilly.org)

The brigade-project-index is interesting because it has fresh GitHub API objects already cached for you, a history of them, and identical data on every other brigade as well. But its data only refreshes every ~hour and pulling from it might be more cumbersome than this application calls for

**Comment by machow** [comment]
Ah, I didn't realize that laddr had a JSON api! Once we have the DB user added for laddr, people could add a task in the pipeline that pulls repo names and saves them somewhere (maybe as a parent task to tap-github somehow?). (or actually I guess could do it now from the json api)

**Comment by themightychris** [comment]
CSV too: https://codeforphilly.org/projects?format=csv

Goes for pretty much everything: https://codeforphilly.org/project-updates?format=json

**Comment by machow** [comment]
@jr-s

**Comment by jaredgk** [comment]
So I've been playing around with this and I have a script that completes steps 1 and 2 (curl to get JSON, python script to parse, sed to edit the yml file) and just got step 3 working (github action to execute script and push file change), I haven't patched them together yet but that should be straightforward. If there's still a need for this, should I open a pull request? (I haven't contributed to an open-source project before so I don't totally know the procedure here)

**Comment by dherbst** [comment]
There is a need, yes!  Please open the pull request and we can work through it.

**Comment by dherbst** [comment]
Closed by #42
